services/data/postgres_async_db.py

```diff
@@ -510,6 +537,17 @@ async def update_heartbeat(self, flow_id: str, run_id: str):
         return DBResponse(response_code=result.response_code,
                           body=json.dumps(body))
 
+    async def update_run_tags(self, flow_id: str, run_id: str, run_tags: list, cur: aiopg.Cursor = None):
+        run_key, run_value = translate_run_key(run_id)
+        filter_dict = {"flow_id": flow_id,
+                       run_key: str(run_value)}
+        set_dict = {"tags": "'" + json.dumps(run_tags) + "'"}  # TODO we can/should do better than manual quoting
```

Comment:
do you want to fix this TODO before merging?

Reply:
Fixed